src/test/pretty/block-disambig.rs

```diff
@@ -7,7 +7,7 @@
 
 use std::cell::Cell;
 
-fn test1() { let val = &0; { } *val; }
+fn test1() { let val = &0; { } let _ = *val; }
```

Comment:
This is an explicit test for that block not to be identified as multiplied by `val`. Disable the lint here.